### PR TITLE
add env var ADDITIONAL_PARAMETERS

### PR DESCRIPTION
**Pull request by gi8lino**

### CODE REVIEW

**Review by rxbn — APPROVED** [review]
LGTM

**Comment by rxbn** [comment]
could you please add a CHANGELOG file similar to the ones from helminator and syncflaer?

**Comment by gi8lino** [comment]
> could you please add a CHANGELOG file similar to the ones from helminator and syncflaer?

done

File: CHANGELOG.md [hunk 1]
```diff
@@ -0,0 +1,148 @@
+# Changelog
+
+## [v2.0.4](https://github.com/containeroo/mirrIO/tree/v2.0.4) (2021-04-03)
```
Comment (rxbn) [inline CHANGELOG.md:3]: since we are adding a new feature, I would suggest bumping the version to v2.1.0

**Review by rxbn — COMMENTED** [review]
have a look at my suggestion

File: CHANGELOG.md (hunk 1 above)
Comment (gi8lino) [reply CHANGELOG.md]: true! I've changed it

**Comment by rxbn** [comment]
could you please also remove the multiarch build and only leave amd64 and add that to the changelog as well? thanks!

**Review by rxbn — APPROVED** [review]
ready to merge!

**Pull request merged**